src/lucky/memoizable.cr

```diff
@@ -0,0 +1,40 @@
+module Lucky::Memoizable
+  # Caches the return value of the method. Helpful for expensive methods that are called
+  # more than once.
```

Comment:
I am fairly certain the docs only take the first line into account for the summary. If that's the case (can check with `crystal docs` then I think this should go on one line